tools/Azure.Mcp.Tools.Sql/src/Options/SqlOptionDefinitions.cs

```diff
@@ -50,4 +56,52 @@ public static class SqlOptionDefinitions
     {
         IsRequired = true
     };
+
+    public static readonly Option<string> AdministratorLoginOption = new(
+        $"--{AdministratorLogin}",
+        "The administrator login name for the SQL server."
+    )
+    {
+        IsRequired = true
```

Comment:
All of these need to be updated, use Required and explicitly set Descrtpion prop.